CondFormats/PCLConfig/interface/AlignPCLThresholds.h

```diff
@@ -0,0 +1,58 @@
+#ifndef _AlignPCLThresholds_h_
+#define _AlignPCLThresholds_h_
```

Comment:
@ghellwig OK, picky but probably safer. 
Is it recommended in the coding rules? I see it's rather common practice troughout the `CondFormats` package to define guards with the name of the class alone.